Development/nmos-cpp-node/node_implementation.cpp

```diff
@@ -766,6 +774,116 @@ void node_implementation_thread(nmos::node_model& model, slog::base_gate& gate_)
     events.wait();
 }
 
+// Example callback to load RSA key and certificate chain
+nmos::load_cert_handler make_node_implementation_load_rsa_handler(nmos::node_model& model, slog::base_gate& gate)
+{
+    // this example loads the RSA key and certificate chain from files for the caller
+    if (model.settings.has_field(impl::fields::rsa))
+    {
+        const auto& rsa = impl::fields::rsa(model.settings);
+        const auto private_key_file = utility::us2s(rsa.at(U("private_key_file")).as_string());
+        const auto certificate_chain_file = utility::us2s(rsa.at(U("certificate_chain_file")).as_string());
```

Comment:
Before finally merging the PR, I suggest adding `field`s for these, in either `impl` or `nmos::experimental` depending on whether the settings/handlers end up as 'examples' or 'defaults' respectively.